templates/course-results/lessons.php

```diff
@@ -159,14 +155,14 @@
 
 					<span class="lesson-grade"><?php echo esc_html( $lesson_grade ); ?></span>
 
-				</h2>
+				<?php echo $course_has_lessons_in_modules ? '</h4>' : '</h3>'; ?>
```

Comment:
"Other Lessons" are never inside a module, so this should always be an `h3` given the header is always an `h2`. Right now it looks like this:

![Screen Shot 2020-06-12 at 7 17 39 AM](https://user-images.githubusercontent.com/1190420/84497465-d5233d00-ac7c-11ea-853a-47ebcc12bba3.jpg)